components/feature/session/src/main/java/mozilla/components/feature/session/behavior/EngineViewBottomBehavior.kt

```diff
@@ -37,8 +38,13 @@ class EngineViewBottomBehavior(
         return super.layoutDependsOn(parent, child, dependency)
     }
 
+    /**
+     * Apply vertical clipping to [EngineView]. This requires [EngineViewBottomBehavior] to be set
+     * in the direct descending child of the [CoordinatorLayout].
```

Comment:
nit: Would add to this: to be set in/on the [EngineView] or its parent. Must be a direct descending child of [CoordinatorLayout].